examples/demo/Standard_Editors/TableEditor_demo.py

```diff
@@ -2,34 +2,37 @@
 #  License: BSD Style.
 
 """
+WARNING
+-------
```

Comment:
This does look pretty prominent... Just wondering if you meant for it to look this big.
![Screenshot 2020-06-26 at 17 29 51](https://user-images.githubusercontent.com/3673984/85879766-d141ff80-b7d2-11ea-90db-9188afaa6b14.png)

Smaller options would be `**WARNING**` (without `----` in the next line), or something like that.

Reply:
I meant it to be drawing attention and the smaller option seemed to blend in a bit too much.

A warning block would be ideal but they don't seem to work with the demo app. Maybe this "definition" pattern draws enough attention:
<img src="https://user-images.githubusercontent.com/25366751/85882902-04d35880-b7d8-11ea-9b1d-f818115e3a15.png" width=300>

Reply:
Yep. That looks like a good compromise.